math/src/field/fields/fft_friendly/stark_252_prime_field.rs

```diff
@@ -50,6 +50,34 @@ impl FieldElement<Stark252PrimeField> {
         }
         bytes
     }
+
+    /// No std version of `to_bytes_be` from `ByteConversion` trait
+    /// This follows the convention used by
+    /// Starkware and Lambdaclass Cairo VM It's the same as ByteConversion to_bytes_be.
+    pub fn to_bytes_be(&self) -> [u8; 32] {
+        let limbs = self.representative().limbs;
+        let mut bytes: [u8; 32] = [0; 32];
+
+        for i in 0..4 {
+            let limb_bytes = limbs[i].to_be_bytes();
+            for j in 0..8 {
+                bytes[i * 8 + j] = limb_bytes[j]
+            }
+        }
+        bytes
+    }
+}
+
+impl PartialOrd for FieldElement<Stark252PrimeField> {
+    fn partial_cmp(&self, other: &Self) -> Option<core::cmp::Ordering> {
+        self.representative().partial_cmp(&other.representative())
+    }
+}
+
+impl Ord for FieldElement<Stark252PrimeField> {
+    fn cmp(&self, other: &Self) -> core::cmp::Ordering {
+        self.representative().cmp(&other.representative())
```

Comment:
Are we sure that cmp is the best way (in terms of performance) to do this? If I do not remember incorrectly, @Oppen found that in some cases this was slow

Reply:
The derive was slow, this should be fine